src/agoradatatools/process.py

```diff
@@ -309,7 +309,7 @@ def process_all_files(
     "LOCAL",
     "--platform",
     "-p",
-    help="Platform that is running the process. Must be one of LOCAL, GITHUB, or NEXTFLOW (Optional).",
+    help="Platform that is running the process. Must be one of LOCAL, GITHUB, or NEXTFLOW (Optional, defaults to LOCAL).",
```

Comment:
For consideration for a different  possible PR: You've changed all "Nextflow" references to "Seqera" instead since the platform got renamed. Do we want to also change this flag from NEXTFLOW to SEQERA, or leave it?

Reply:
I think it's ok because it is running on Seqera Platform, but it is a Nextflow workflow.